cpp/ql/src/semmle/code/cpp/PrintAST.qll

```diff
@@ -154,35 +147,32 @@ class PrintASTNode extends TPrintASTNode {
   /**
    * Gets the `Function` that contains this node.
    */
-  private Function getEnclosingFunction() {
-    result = getParent*().(FunctionNode).getFunction()
-  }
+  private Function getEnclosingFunction() { result = getParent*().(FunctionNode).getFunction() }
 }
 
+/**
+ * Retrieves the canonical QL class for entity `el`
+ */
+private string qlClass(ElementBase el) { result = "[" + el.getCanonicalQLClass() + "]: " }
```

Comment:
```suggestion
private string qlClass(ElementBase el) { result = "[" + el.getCanonicalQLClass() + "] " }
```
Minor nit, but I think the colon is unnecessary and makes lines that are already prefixed by, say, `0:` look odd with a second colon.

Reply:
Done and done.

Reply:
In fact, I'll change the expression to `"[" + concat(el.getCanonicalQLClass(), ",") + "] "`, just in case multiple `getCanonicalQLClass()` values were to crop up.